docs/src/main/tut/typeclasses/eq.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: docs
+title:  "Eq"
+section: "typeclasses"
+source: "kernel/src/main/scala/cats/kernel/Eq.scala"
+scaladoc: "#cats.kernel.Eq"
+---
+
+# Eq
+
+Show is an alternative to the standard Java `equals` method.
+It is defined by the single method `eqv`:
+
+```scala
+def eqv(x: A, y: A): Boolean
+```
+
+In Scala it's possible to compare any two values using `==` (which desugars to Java `equals`).
+This is because `equals` type signature uses `Any` (Java's `Object`) to compare two values.
+This means that we can compare two completely unrelated types without getting a compiler error.
+The Scala compiler may warn us in some cases, but not all, which can lead to some weird bugs.
+For example this code will raise a warning at compile time:
+
+
+```tut:book:fail
+42 == "Hello"
+```
+
+While this code will compile without a hitch:
+
+```tut:book
+"Hello" == 42
+```
+
+Ideally, Scala shouldn't let us compare two types that can never be equal.
+
+As you can probably see in the type signature of `eqv`, it is impossible to compare two values of different types,
+eliminating these types of bugs altogether.
+
+The `Eq` syntax package also offers some handy symbolic operators:
+
+```tut:book
+import cats.implicits._
+
+1 === 1
+
+"Hello" =!= "World"
+```
+
+Implementing `Eq` instances yourself for every data type might seem like huge drawback compared to only slight gains of typesafety.
```

Comment:
not that I know. We're using it in a project since a few weeks and so far so good.

That said, I'm far from being an expert :)